source/agora/common/Block.d

```diff
@@ -95,3 +95,78 @@ public struct Block
     ///
     public Transaction tx;
 }
+
+/*******************************************************************************
+
+    Creates the genesis block.
+    Note: The output specifies a randomly-generated address,
+    later to be replaced with proper outputs.
+
+    Returns:
+        the genesis block
+
+*******************************************************************************/
+
+public Block getGenesisBlock ( )
```

Comment:
```suggestion
public Block getGenesisBlock ()
```

Reply:
Ah, makes it easier to grep? Add it to the style guide!

Reply:
It already is there: https://github.com/bpfkorea/agora/pull/74/files#diff-f69cd4b5588d09bb541f85596fdacad3R57